src/main/java/stringcalculator/Operand.java

```diff
@@ -0,0 +1,27 @@
+package stringcalculator;
+
+import java.math.BigInteger;
+
+public class Operand {
+	private static final String OPERAND_MUST_NOT_BE_NEGATIVE = "피연산자는 음수가 될 수 없습니다.";
+
+	private static final String INVALID_FORMATTED_OPERAND = "잘못된 형식의 피연산자 입니다.";
+
+	private final BigInteger value;
+
+	public Operand(String value) {
+		try {
+			this.value = new BigInteger(value);
+
+			if (this.value.compareTo(BigInteger.ZERO) < 0) {
+				throw new RuntimeException(OPERAND_MUST_NOT_BE_NEGATIVE);
+			}
+		} catch (NumberFormatException e) {
+			throw new RuntimeException(INVALID_FORMATTED_OPERAND, e);
+		}
+	}
```

Comment:
아래와 같이 주생성자는 인스턴스 변수와 동일한 타입의 생성자 파라미터를 받아 초기화하는 생성자를 두고,  문자열을 받아서 Operand 객체를 생성하는 것은 [정적 팩토리 메서드](https://tecoble.techcourse.co.kr/post/2020-05-26-static-factory-method/)를 활용해볼 수 있을 것 같아요. 🙂

```suggestion
    public Operand(BigInteger value) {
        if (value.compareTo(BigInteger.ZERO) < 0) {
            throw new RuntimeException(OPERAND_MUST_NOT_BE_NEGATIVE);
        }

        this.value = value;
    }

    public static Operand of(String value) {
        try {
            return new Operand(new BigInteger(value));
        } catch (NumberFormatException e) {
            throw new RuntimeException(INVALID_FORMATTED_OPERAND, e);
        }
    }
```

Reply:
반영 해 보겠습니다.